app/assets/stylesheets/home.scss

```diff
@@ -1,16 +1,121 @@
+$laptop-width: 969px;
+
 #home-banner {
   width: 100%;
   height: 600px;
-  background: url(http://goo.gl/MGh98r);
-  background-size: cover;
+  margin-bottom: 200px;
+  background: image-url('home/banner-bg.jpg');
+  background-size:cover;
+  position: relative;
+
+  @media only screen and (max-width: $medium-screen) {
+    height: 400px;
+    margin-bottom: 150px;
+  }
+
 }
 
+#laptop {
+  position: absolute;
+  bottom: -175px;
+  max-height: 650px;
+  height: 100%;
+  width: 100%;
+  text-align:center;
+
+  @media only screen and (max-width: $medium-screen) {
+    bottom:-120px;
+  }
+
+  #laptop-bg-container {
+    position: relative;
+    z-index:2;
+    bottom: 0px;
+
+    @media only screen and (max-width: $laptop-width) {
+      position: absolute;
+      bottom: 0px;
+    }
+  }
+
+  #laptop-bg {
+    height: auto;
+    width: 100%;
+    max-width: $laptop-width;
+  }
+
+  #laptop-content-container {
+    position: absolute;
+    width: 100%;
+    text-align:center;
+    padding-top: 25px;
+    z-index: 0;
+
+    @media only screen and (max-width: $large-screen) {
+      padding-top: 2.3%;
+    }
+
+    @media only screen and (max-width: $laptop-width) {
+      padding-top: 0px;
+      padding-bottom: 6.5%;
+      position: absolute;
+      bottom: 0px;
+    }
```

Comment:
Yay for responsiveness! If you put in `$laptop-width` in [_grid_settings.scss](https://github.com/calblueprint/calblueprint.org/blob/master/app/assets/stylesheets/vendor/base/_grid-settings.scss) then you can rewrite your media queries as

    @include media($large-screen) { ... }
    // or
    @include media($laptop-width) { ... }

which is a little nicer.